tests/bootstrap.php

```diff
@@ -401,6 +401,82 @@ function loadDataset()
                 'end_date'       => '2016-12-31',
                 'entities_id'     => '_test_root_entity'
             ]
+        ], 'Problem' => [
```

Comment:
You have to update the `_version` of dataset (line 113).

Reply:
done